### DISCUSSION TITLE
BWG 1.3.0 ALPHA Patch Notes

### DISCUSSION THREAD

**Discussion by AOCAWOL** [CorgiTaco/BYG#1324] [BWG Patch Notes]
![1 3 0](https://github.com/Potion-Studios/BYG/assets/51396466/c1c84ec7-dd6d-47cb-a261-116cdaae689b)

We're back with our FINAL alpha for BWG! Almost all of the biomes and content is finalized and ready for launch! We are finishing up some super exciting new content and revamps for the existing biomes we still need to port. We've got lots to cover so lets dig into it!

Just a reminder you need to be a paid patron to access this Alpha build. You can become a patron at 'patreon.com/biomesyougo'
The goal of this Alpha is for players to test out our new features and give feedback on them and bug reporting.

**Some things to know for this Alpha:**
-Not all of BWG's biomes are in this alpha but by release, we will have them all (excluding nether & end)
-Some of our features are WIP so they might be buggy or not fully finished

### BIOMES
Biome Changes:
-Eroded Borealis has had some changes to the generation and colors
-Giant boulders have been added to various biomes such as the Windswept Desert
-The Current generation in the Basalt Barrera, Lush Stacks, Pumpkin Patch, and Dead Sea is not finalized and will be by beta!
-Increased vegetation and tree variety in the Tropical Rainforest

This alpha currently contains the following biomes:
_-Allium Shrublands {REWORKED}
-Amaranth Grassland
-Araucaria Savanna
-Aspen Boreal
-Atacama Outback
-Baobab Savanna
-Basalt Barrera
-Bayou
-Black Forest
-Canadian Shield
-Cika Woods
-Coconino Meadow
-Coniferous Forest
-Crimson Tundra {NEW}
-Cypress Swamplands
-Dacite Ridges
-Dacite Shore
-Dead Sea 
-Ebony Woods
-Enchanted Tangle {REVAMPED}
-Eroded Borealis {NEW}
-Firecracker Chaparral
-Fragment Jungle {REVAMPED}
-Frosted Taiga
-Howling Peaks
-Ironwood Gour {NEW}
-Jacaranda Jungle
-Lush Stacks
-Maple Taiga 
-Mojave Desert
-Orchard
-Prairie
-Pumpkin Valley
-Rainbow Beach
-Red Rock Valley
-Redwood Thicket
-Rose Fields
-Rugged Badlands {NEW}
-Sakura Grove [REVAMPED]
-Sierra Badlands
-Skyris Vale
-Temperate Grove
-Tropical Rainforest
-Weeping Witch Forest
-White Mangrove Marshes
-Windswept Desert
-Zelkova Forest_

**FRAGMENT JUNGLE - REVAMPED BIOME**
![2024-07-09_21 50 49](https://github.com/Potion-Studios/BYG/assets/51396466/74ab63e6-4b59-48b3-add4-bfef1fe4c2eb)

The Fragment Jungle has received HUGE overhauls its practically a brand new biome! This biome is inspired by the Zhangjiajie Mountains in China. This biome was so much fun to redo and we hope you enjoy its vibrant life.

NOTES: 
-Vegetation isn't generating on the tops of spikes as intended and will be addressed by beta

**ENCHANTED TANGLE - REVAMPED BIOME**
![2024-07-09_21 42 38](https://github.com/Potion-Studios/BYG/assets/51396466/0a243eb9-18ce-4d87-a525-814b234847e9)

Speaking of revamps, the Enchanted Tangle has received huge overhauls to it's functionality and magical essence. We are adding a new utility/revamp to Imbued Enchanted Wood (Green or Blue). We know that some players don't like our use of amethyst in this biome and we encourage you to either disable this biome or datapack it out. We are implementing measures to make it as fair as we see fit.

NOTES:
-Imbued Wood on Beta will be obtainable for the player to use however it will not maintain it's crystal-growing properties if placed by a player
-Enchanted Saplings will NOT grow crystal & glowberry bearing trees on Beta release

### STRUCTURES

**VILLAGES - RE-ADDED STRUCTURES**
![2024-07-09_21 31 39](https://github.com/Potion-Studios/BYG/assets/51396466/b5d2cdf7-dff4-40a7-914d-ad7e6e3cfd38)

BWG has re-added it's villages. Players can discover rich and vibrant villages in the Skyris Vale, Weeping Witch Forest, Redrock Valley, and Pumpkin Valley

### MOBS

-Slight texture modification & new idle animations for pumpkin warden

**ODDION REVAMP**
![2024-07-09_21 47 52](https://github.com/Potion-Studios/BYG/assets/51396466/258945bb-3f1e-4372-aa0f-6e36359ba88a)

On the art side of things I wasn't super pleased with the Oddion artwork and wanted to revamp it for final release. He also has new animations per community suggestion so that he feels more alive especially when idle.

###BUG FIXES & QOL UPDATES

-Skyris, Salem, Pumpkin Patch, and Red Rock Villages have new custom processors
-Added French, Simplified Chinese, Brazilian Portuguese
-Added Sniffer Compat
-Re-Added BWG Villager Professions
-Added Fuels
-Added Villager Types for BWGBiomes
-Added Dispenser Behavior for BWG Items
-Made all Saplings Work
-Added non Wood saplings
-Made more blocks pottable
-Added WTHIT Compat plugin for Plant Growth
-Improve Fall in QuickSand Advancement and make it obtainable in red quicksand
-Added BlueBerry Bush to Bees Growables Tag

-Fixed Aspen Manor 1 pool being called for 2
-Added BWG bookshelves to EnchantmentPowerProvider Tag to fix our bookshelves not working with enchantment tables
-Added BWG Grasses and Vines to Vanilla Replaceable tag
-Removed Nether and End Content
-Removed Birch Trees from Sakura Grove
-Fixed Maple Door RenderType

### FINAL NOTES
We are getting so close to Beta release for the public. We appreciate your feedback, patience, and excitement for this release!

![official_promo](https://github.com/Potion-Studios/BYG/assets/51396466/3e371aa7-9706-481c-a6c8-42023d2cd623)

**Comment by adamiola** [comment]
When is open beta releasing?

**Comment by Subthy** [comment]
gimme gimme beta

**Comment by marlester-dev** [comment]
I am very excited!
